website/docs/Threshold Decryption.md

```diff
@@ -0,0 +1,79 @@
+---
+slug: Threshold Decryption.md
+sidebar_position: 16
+---
```

Comment:
Should this content move to CBD.md?

Reply:
yes, I've kept it around for my own reference for now :)